src/api/utils/related.py

```
@@ -44,11 +52,14 @@ def get_related_project(model):
 
     for f in model._meta.get_fields():
         if isinstance(f, ForeignKey):
-            rel_obj = getattr(model, f.name)
+            try:
+                rel_obj = getattr(model, f.name)
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)